sdk/inc/azure/core/internal/az_precondition_internal.h

```diff
@@ -51,7 +51,7 @@ az_precondition_failed_fn az_precondition_failed_get_callback();
 // earlier without verifying, because if __analysis_assume() is not available on earlier compiler
 // version, there will be a compilation error.
 // For more info, see
-// https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/using-the--analysis-assume-function-to-suppress-false-defects
```

Comment:
I was going to ask why link verification didn't catch this, but guessing that since these are just links within doc comments, not an actual hyperlink within markdown or other such docs, it doesn't get validated.

@danieljurek, @weshaggard has this come up as a way to improve link verification in other languages? If we have typos in links within source code comments that could 404, should those be verified?

Reply:
You are correct that we don't validate random links in any text we only look at well formed links in MD or html. This has come up in .NET XML doc comments recently, https://github.com/Azure/azure-sdk-for-net/issues/22216, although we don't have a solution for that currently either. It can be tricky to parse out links in random source code.

Reply:
That's what I figured. Thanks!

Reply:
#1980 Is about this, but based on @weshaggard's response, I think we can close it?